llmfoundry/models/layers/attention.py

```diff
@@ -600,20 +626,29 @@ def get_qkv(
                 dtype = query.dtype
                 query = self.q_ln(query).to(dtype).view(q_shape)
             return query, key, value
+        elif self.fused_qkv:
```

Comment:
this should just be separate if statement i think since the above if statement returns

Reply:
done